data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/test_modify_unit.cfg

```diff
@@ -15,7 +15,7 @@
 # have the side-effect of moving it to the correct side's recall list.
 
 #####
-# API(s) being tested: [modify_unit],[role][auto_recall]
+# API(s) being tested: [role][auto_recall]
```

Comment:
This test was added as part of fixing a bug in `[modify_unit]`, it should definitely be in this list.

Reply:
I'd say this should just be `[modify_unit]` then.

Reply:
Changed to `[modify_unit]`.